### PR TITLE
chore: extracting another part from `add_earn` refactoring

### PR DESCRIPTION
**Pull request by nicolasburtey**
this mostly include nome renaming from payerWalletId and walletId to senderWallerId. there are more renaming from payer to sender to do on the codebase, but I just had extract the one I had encounter doing the add_earn use case refactoring

### CODE REVIEW

File: src/app/wallets/index.types.d.ts [hunk 1]
```diff
@@ -47,31 +48,31 @@ type GetOnChainFeeByWalletIdArgs = {
...
   logger: Logger
 }
 
 type PayLnInvoiceByWalletIdArgs = {
-  walletId: WalletId
+  senderWalletId: WalletId
   paymentRequest: EncodedPaymentRequest
   memo: string | null
-  userId: UserId
+  payerUserId: UserId
```
Comment (nicolasburtey) [inline src/app/wallets/index.types.d.ts:60]: note: will convert to senderUserId more broadly in another PR

File: src/graphql/root/mutation/intraledger-payment-send.ts [hunk 2]
```diff
@@ -1,17 +1,17 @@
-import { Accounts, Wallets } from "@app"
+import { getUsernameFromWalletId } from "@app/accounts"
+import { getWallet, intraledgerPaymentSendUsername } from "@app/wallets"
```
Comment (dolcalmi) [inline src/graphql/root/mutation/intraledger-payment-send.ts:2]: please use previous version

```js
import { Accounts, Wallets } from "@app"
```

File: src/core/on-chain/index.ts [hunk 3]
```diff
@@ -1,4 +1,5 @@
-import { Wallets, Prices } from "@app"
+import { Prices, Wallets } from "@app"
+import { getUser } from "@app/users"
```
Comment (dolcalmi) [inline src/core/on-chain/index.ts:2]: please `Users` from `@app`

File: test/integration/02-user-wallet/02-send-lightning.spec.ts [hunk 4]
```diff
@@ -1,9 +1,23 @@
-import { createHash, randomBytes } from "crypto"
+import { Wallets } from "@app"
+import { intraledgerPaymentSendUsername } from "@app/wallets"
```
Comment (dolcalmi) [inline test/integration/02-user-wallet/02-send-lightning.spec.ts:2]: use method from `Wallets`

**Review by dolcalmi — CHANGES_REQUESTED** [review]

File: test/integration/02-user-wallet/02-send-lightning.spec.ts (hunk 4 above)
Comment (nicolasburtey) [reply test/integration/02-user-wallet/02-send-lightning.spec.ts:2]: do we want to use the line 1 format here or line 2?

**Review by dolcalmi — APPROVED** [review]

**Pull request merged**